pkg/git/ssh_config.go

```diff
@@ -67,10 +57,31 @@ func AddSSHConfig(cfg config.PipedGit) error {
 		}
 		cfgPath = path.Join(home, ".ssh", "config")
 	}
-	dir := filepath.Dir(cfgPath)
+	sshDir := filepath.Dir(cfgPath)
+
+	if err := os.MkdirAll(sshDir, 0700); err != nil {
+		return fmt.Errorf("failed to create a directory %s: %v", sshDir, err)
+	}
 
-	if err := os.MkdirAll(dir, 0700); err != nil {
-		return fmt.Errorf("failed to create a directory %s: %v", dir, err)
+	var sshKeyFile string
+	if cfg.SSHKeyFile != "" {
+		f, err := os.CreateTemp(sshDir, "piped-ssh-key-*")
+		if err != nil {
+			return err
+		}
```

Comment:
Across the docs looks like we should close/remove the opened temp file by ourselves 👀  Should we?

> It is the caller's responsibility to remove the file when it is no longer needed.

ref: [docs](https://golang.org/pkg/os/#CreateTemp)

Reply:
You are right. Let me add a TODO to remove the added file.

Reply:
🙆‍♀️

Reply:
@khanhtc1202 Added.